label_studio/data_manager/api.py

```diff
@@ -100,6 +100,17 @@ def reset(self, _request):
         queryset.all().delete()
         return Response(status=204)
 
+    @staticmethod
+    def evaluate_predictions(tasks):
```

Comment:
It's a cool feature but degrades UI performance a lot. we can do batch prediction instead, in later releases

Reply:
It will work only with paginated tasks, like batch predictions. You can disable it via Settings => Machine Learning => Display ML-predicted annotations when labeling. It's temporary workaround. 

I have an idea to implement actions for selected tasks - "Retrieve ML predictions" & "Delete predictions" & "Convert predictions to annotations" (the last one was in LS 0.9).